RiotSharp/Interfaces/IRiotApi.cs

```diff
@@ -209,74 +208,98 @@ public interface IRiotApi
 
         #region Match
         /// <summary>
+        /// Get the matches' ID of the specified tournament synchronously.
+        /// </summary>
+        /// <param name="region">Region in which the tournament took place.</param>
+        /// <param name="tournamentCode">The tournament ID to be retrieved.</param>
+        /// <returns>A list containing the matches' ID.</returns>
+        List<long> GetMatchIdsByTournamentCode(Region region, string tournamentCode);
+
+        /// <summary>
+        /// Get the matches' ID of the specified tournament asynchronously.
+        /// </summary>
+        /// <param name="region">Region in which the tournament took place.</param>
+        /// <param name="tournamentCode">The tournament ID to be retrieved.</param>
+        /// <returns>A list containing the matches' ID.</returns>
+        Task<List<long>> GetMatchIdsByTournamentCodeAsync(Region region, string tournamentCode);
+
+        /// <summary>
         /// Get match information about a specific match synchronously.
         /// </summary>
         /// <param name="region">Region in which the match took place.</param>
         /// <param name="matchId">The match ID to be retrieved.</param>
-        /// <param name="includeTimeline">Whether or not to include timeline information.</param>
-        /// <returns>A match detail object containing information about the match.</returns>
-        MatchDetail GetMatch(Region region, long matchId, bool includeTimeline = false);
+        /// <param name="accountId">If provided, used to identify the participant to be unobfuscated.</param>
+        /// <returns>A match object containing information about the match.</returns>
+        Match GetMatch(Region region, long matchId, long? accountId = null);
```

Comment:
As far as I know Riot removed the `accountId` parameter. Ref: https://developer.riotgames.com/api-methods/#match-v3/GET_getMatch